bin/ch-build

```diff
@@ -1,6 +1,12 @@
 #!/bin/sh
 
-. $(dirname "$0")/base.sh
+libexec_base=$(dirname "$0")
```

Comment:
* Can we call it `LIBEXEC` for shortness and consistency with upper-case variables elsewhere in the scripts?
* Let's use the longer voodoo currently used for `CH_BIN`, since it's more robust to symlinks and other stuff I've forgotten.

Reply:
Both good points, I'll adapt!